app/bundles/CoreBundle/Helper/InputHelper.php

```diff
@@ -443,16 +443,13 @@ function ($matches) {
             // detect if there is any unicode character in the passed string
             $hasUnicode = strlen($value) != strlen(utf8_decode($value));
 
-            // Encode the incoming value before cleaning, it convert unicode to encoded strings
-            $value = $hasUnicode ? rawurlencode($value) : $value;
-
-            $value = self::getFilter(true)->clean($value, 'html');
+            $value = self::getFilter(true)->clean($value, $hasUnicode ? 'raw' : 'html');
 
             // After cleaning encode the value
             $value = $hasUnicode ? rawurldecode($value) : $value;
```

Comment:
@kuzmany Should this be removed as well since its been removed above.

Reply:
Don't understand question

Reply:
@kuzmany If I understand well, then @MarketSmart mentioned that these too lines 448, 449 in the white sections are unnecessary:
// After cleaning encode the value
$value = $hasUnicode ? rawurldecode($value) : $value;

Reply:
It's diff, red lines are removed and green lines are added.

Reply:
@kuzmany Yes, you are right. @MarketSmart - by my opinion - mentioned line 447 (red) and line (452) white which are the same. The line  "$value = $hasUnicode ? rawurldecode($value) : $value;" was duplicated in the original code. It really needs or typo?